proposals/0282-record-dot-syntax.rst

```diff
@@ -68,15 +69,25 @@ For the specification we focus on the changes to the parsing rules, and
 the desugaring, with the belief the type checking and renamer changes
 required are an unambiguous consequences of those.
 
-2.1 ``RecordDotSyntax`` language extension
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+2.1 Language extensions
+~~~~~~~~~~~~~~~~~~~~~~~
 
-This change adds a new language extension ``RecordDotSyntax``.
+This change adds new language extensions ``OverloadedRecordDot`` and
+``OverloadedRecordUpdate``.
+
+The effect of ``OverloadedRecordDot`` is to:
+  - alllow '``.``' in record exprssions;
+  - desguar field access via '``.``'to ``getField`` expressions;
+The effect of ``OverloadedRecordUpdate`` is to:
+  - desguar record updates to ``setField`` expressions.
+
+A record update expression like ``e{lbl₁.lbl₂ = val}`` requires both
```

Comment:
That is, `OverloadedRecordUpdate` makes no sense without `OverloadedRecordDot`?   Is that idea that the former will silently be a no-op without the lattter?  (That seems undesirable)

Reply:
- No, `OverloadedRecordUpdate` makes sense without `OverloadedRecordDot`;
  - It means that record updates will desugar to `setField` expressions;
  - It means that "nested" updates will be syntax errors (`.` is not allowed);
- I have added a paragraph to clarify what happens in these two cases (section 2.1).